logging.rst

```diff
@@ -32,6 +32,15 @@ To log a message, inject the default logger in your controller or service::
         $logger->info('I just got the logger');
         $logger->error('An error occurred');
 
+        // log messages can also contain placeholders (wrap the placeholder name with braces)
+        // these placeholders are useful for logging tools, which can aggregate log messages
+        // that are the same except for some variable values inside them, for translation
+        // systems in order to create localized messages and for security because escaping
+        // will be done by the implementation
```

Comment:
```suggestion
        // systems in order to create localized messages, and for security because escaping
        // can then be done by the implementation in a context-aware fashion.
```

Adding the Oxford Comma (it matters here), and and expanding the wording slightly.